python/ray/experimental/workflow/api.py

```diff
@@ -30,18 +26,16 @@ def step(func) -> WorkflowStepFunction:
 
 # TODO(suquark): Raise an error when calling run() on an existing workflow.
 # Maybe we can also add a run_or_resume() call.
-# TODO(suquark): Rename "workflow_root_dir" to "storage_url".
 
 
-def run(entry_workflow: Workflow, workflow_root_dir=None,
+def run(entry_workflow: Workflow, storage_url=None,
```

Comment:
Could we just make this `storage: Union[str, Storage] = None` instead? Then you can either pass in a custom storage implementation, or a s3/filesystem uri.